backend/project/endpoints/submissions/submissions.py

```diff
@@ -58,6 +60,12 @@ def get(self, uid=None) -> dict[str, any]:
                     return data, 400
                 filters["project_id"] = int(project_id)
 
+            filters = {
+                key: value for key, value
+                in filters.items()
+                if key in {f.name for f in fields(Submission)}
```

Comment:
This can contain other fields than columns, use 

```py
if key in model.__table__.columns
```

Reply:
https://github.com/SELab-2/UGent-3/pull/376/commits/9194ae0199d3c8ce4906ed3f0c0812446d04deac